doc/content/gateways/COTX-X1 Lite gateway

```diff
@@ -0,0 +1,47 @@
+---
+title: "COTX X1 Lite"
+description: ""
+---
+
+COTX X1 Lite is a standard LoRaWAN indoor gateway and a Helium data-only hotspot.
+It is highly compatible and has a user-friendly interface.
+Rich in software tools, powerful back office management and technical support.
+Compact and exquisite appearance, easy to install and deploy.
+![Lite-1.png](./TTN-X1%20Lite-1.png)
```

Comment:
Please, don't use such file names. Change this image name to "x1-lite.png" or something similar. Also, this is not the way that we add images to our doc. Instead, we use the `figure` shortcode. Read our [CONTRIBUTING to docs guide](https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/CONTRIBUTING.md).